crates/wasmlink/tests/run.rs

```diff
@@ -28,6 +28,7 @@ fn wasmlink_file_tests() -> Result<()> {
         let entry = entry?;
 
         let path = entry.path();
+        println!("{:?}", path);
```

Comment:
This one was actually intentional but I can expand it, otherwise when this fails it has no indication as to what test was actually being run, it just fails with an opaque error message about the internals of the wit/wasm/etc.